### ISSUE TITLE
Why PanEnded Event triggers on Click Event in MainPanel

### ISSUE THREAD

**Issue by mukeshbadgujar** [amcharts/amcharts5#579]
Please check for the **bug**:
why panended event is calling on clicking on chart.

Steps to Generate Issue:
1. Click on chart
you will see click event alert
and later you see panended alert 

```
mainPanel.events.on("panended", function() {
	alert("PanEnded")
});

mainPanel.events.on("click", function() {
	alert("click")
});
```
Code: https://jsfiddle.net/mukeshbadgujar/0uz5c9kt/6/

**Comment by martynasma** [comment]
We'll add a `pancanceled` event in the next release.

It will kick in if pointer hasn't moved between `panstarted` and `panended`.

Skipping `panended` event altogether would be wrong, because `panstarted` event always kicks in on pointerdown.

Also, I strongly suggest adding `click` event on `mainPanel.plotContainer`, rather than panel itself, as plot container is interactive element, not the panel/chart itself.

**Comment by martynasma** [comment]
Implemented in [5.2.19](https://github.com/amcharts/amcharts5/releases/tag/5.2.19).

## [5.2.19] - 2022-08-26

### Added
- `geoPoint()` method added to `MapChart`. Returns a geoPoint of the current the center of the map viewport.
- `pancancelled` event added to `XYChart`. It will kick in if pointer is pressed down and released without moving. `panended` event will still kick in.
- `originalEvent` added to `panstarted`, `panended`, and `pancancelled` events.

### Fixed
- Tooltips were not working properly on an `XYChart` if they were placed in a separate root element.
- If an `XYChart` with a `CategoryAxis` had series with missing values, zooming to the part of a series with no data caused value axis not to zoom according to the selection.
- HTML-based tooltips were leaving a visible (albeit transparent) DOM element after tooltip was hidden.
- Clear and erase buttons were not clearing/erasing all the drawings of StockChart properly.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
